plugins/greatexpectations/flytekitplugins/greatexpectations/task.py

```diff
@@ -0,0 +1,239 @@
+import datetime
+import logging
+import os
+import shutil
+import typing
+from dataclasses import dataclass
+
+import great_expectations as ge
+from great_expectations.checkpoint import SimpleCheckpoint
+from great_expectations.core.run_identifier import RunIdentifier
+from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.exceptions import ValidationError
+
+from flytekit import PythonInstanceTask
+from flytekit.core.context_manager import FlyteContext
+from flytekit.extend import Interface
+from flytekit.types.file.file import FlyteFile
+from flytekit.types.schema import FlyteSchema
+
+T = typing.TypeVar("T")
+
+
+@dataclass
+class BatchRequestConfig(object):
+    """
+    Use this configuration to configure Batch Request. A BatchRequest can either be
+    a simple BatchRequest or a RuntimeBatchRequest.
+
+    Args:
+        data_connector_query: query to request data batch
+        runtime_parameters: parameters to be passed at runtime
+        batch_identifiers: identifiers to identify the data batch
+        batch_spec_passthrough: reader method if your file doesn't have an extension
+        limit: number of data points to fetch
+    """
+
+    data_connector_query: typing.Optional[typing.Dict[str, typing.Any]] = None
+    runtime_parameters: typing.Optional[typing.Dict[str, str]] = None
+    batch_identifiers: typing.Optional[typing.Dict[str, str]] = None
+    batch_spec_passthrough: typing.Optional[typing.Dict[str, typing.Any]] = None
+    limit: typing.Optional[int] = None
+
+
+class GETask(PythonInstanceTask[BatchRequestConfig]):
+    """
+    This task can be used to validate your data.
+
+    Args:
+        name: name of the task
+        data_source: tell where your data lives and how to get it
+        expectation_suite: suite which consists of the data expectations
+        data_connector: connector to identify data batches
+        inputs: inputs to pass to the execute() method
+        local_file_path: dataset file path useful for FlyteFile and FlyteSchema
+        checkpoint_params: optional SimpleCheckpoint parameters
+        task_config: batchrequest config
+        data_context: directory in which GE's configuration resides
+
+    TODO: Connect Data Docs to Flyte Console.
+    """
+
+    _TASK_TYPE = "greatexpectations"
```

Comment:
Could we amend this to `great_expectations`?

Reply:
Done.